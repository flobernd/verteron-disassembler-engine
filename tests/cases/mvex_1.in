-64 62 E1 89 14 59 8D DF 4B 4A 36 62 D1 21 1B DB