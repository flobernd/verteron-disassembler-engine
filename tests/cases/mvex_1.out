== [    BASIC ] ============================================================================================
   MNEMONIC: vmulpd [ENC: MVEX, MAP: 0F, OPC: 0x59]
     LENGTH: 10
        SSZ: 64
       EOSZ: 64
       EASZ: 64
   CATEGORY: KNC
    ISA-SET: KNCE
    ISA-EXT: KNCE
 EXCEPTIONS: NONE
 ATTRIBUTES: HAS_MODRM HAS_MVEX 

== [ OPERANDS ] ============================================================================================
##       TYPE  VISIBILITY  ACTION      ENCODING   SIZE  NELEM  ELEMSZ  ELEMTYPE                        VALUE
--  ---------  ----------  ------  ------------   ----  -----  ------  --------  ---------------------------
 0   REGISTER    EXPLICIT     RCW     MODRM_REG    512      8      64   FLOAT64                        zmm17
 1   REGISTER    EXPLICIT       R          MASK     16     16       1       INT                           k4
 2   REGISTER    EXPLICIT       R        NDSNDD    512      8      64   FLOAT64                        zmm30
 3     MEMORY    EXPLICIT       R      MODRM_RM     64      1      64   FLOAT64  TYPE  =                 MEM
                                                                                 SEG   =                  ss
                                                                                 BASE  =                 rbp
                                                                                 INDEX =                none
                                                                                 SCALE =                   0
                                                                                 DISP  =  0x00000000364A4BDF
--  ---------  ----------  ------  ------------   ----  -----  ------  --------  ---------------------------

== [      AVX ] ============================================================================================
  VECTORLEN: 512
  BROADCAST: 1_TO_8
   ROUNDING: DEFAULT
        SAE: N
       MASK: k4 [MERGING]
         EH: N
    SWIZZLE: NONE
    CONVERT: NONE

== [      ATT ] ============================================================================================
   ABSOLUTE: vmulpd 0x364A4BDF(%rbp) {1to8}, %zmm30, %zmm17 {%k4}
   RELATIVE: vmulpd 0x364A4BDF(%rbp) {1to8}, %zmm30, %zmm17 {%k4}

== [    INTEL ] ============================================================================================
   ABSOLUTE: vmulpd zmm17 {k4}, zmm30, qword ptr ss:[rbp+0x364A4BDF] {1to8}
   RELATIVE: vmulpd zmm17 {k4}, zmm30, qword ptr ss:[rbp+0x364A4BDF] {1to8}

== [ SEGMENTS ] ============================================================================================
62 E1 89 14 59 8D DF 4B 4A 36 
:           :  :  :..DISP
:           :  :..MODRM
:           :..OPCODE
:..MVEX
